src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Figure[] value = Figure.values();
+        int figure = random.nextInt(FIGURE_COUNT);
+        return value[figure];

```

Comment:
You have to write the logic for creating a random figure here. Not the return of a random Enum value. Exactly the object.